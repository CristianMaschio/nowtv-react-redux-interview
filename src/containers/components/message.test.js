import React from 'react'
import { shallow } from 'enzyme'

import Message from './message'

describe('Test: <Message />', () => {
  it('renders correctly', () => {
    const props = {
      message: {
        id: '22a454b1-6659-400f-aa77-eaae34a77118',
        userId: '205c9b7a-345d-4aa2-9e4c-56f05f72bbe8',
        message:
          'Aenean auctor gravida sem. Praesent id massa id nisl venenatis lacinia.',
        timestamp: '2016-04-14T16:53:55Z'
      },
      user: {
        id: '205c9b7a-345d-4aa2-9e4c-56f05f72bbe8',
        firstName: 'Jeremy',
        lastName: 'Gomez',
        email: 'jgomezh@cdc.gov',
        avatar: 'http://dummyimage.com/100x100.jpg/dddddd/000000',
        ip: '153.52.102.218'
      }
    }
    const tree = shallow(<Message {...props} />)
    expect(tree).toMatchSnapshot()
  })
})
