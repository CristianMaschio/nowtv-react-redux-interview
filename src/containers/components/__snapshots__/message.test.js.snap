// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test: <Message /> renders correctly 1`] = `
<div
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <div
    className="message-container"
  >
    <div
      className="message-detail "
    >
      <span>
        jgomezh@cdc.gov
      </span>
    </div>
    <img
      alt="user avatar"
      className="avatar"
      src="http://dummyimage.com/100x100.jpg/dddddd/000000"
    />
    <div
      className="message-text"
    >
      Aenean auctor gravida sem. Praesent id massa id nisl venenatis lacinia.
    </div>
  </div>
</div>
`;

exports[`Test: <Message/> renders correctly 1`] = `
<div
  className="home"
>
  <img
    alt="Now TV logo"
    className="logo"
    src="logo.png"
  />
  <AutoSizer
    disableHeight={true}
    disableWidth={false}
    onResize={[Function]}
    style={Object {}}
  >
    <Component />
  </AutoSizer>
</div>
`;

exports[`renders correctly 1`] = `
<div>
  This is Test
</div>
`;
